build.gradle.kts

```diff
@@ -2,22 +2,19 @@ import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 import org.jetbrains.kotlin.gradle.targets.native.tasks.KotlinNativeTest
 
 plugins {
-    kotlin("multiplatform") version "1.4.10"
-    kotlin("plugin.serialization") version "1.4.10"
+    kotlin("multiplatform") version "1.4.31"
+    kotlin("plugin.serialization") version "1.4.30"
     `maven-publish`
 }
 
 allprojects {
     group = "fr.acinq.lightning"
-    version = "1.0-beta11"
+    version = (project.findProperty("publishVersion") as? String?) ?: "snapshot"
 
```

Comment:
We can just keep `version = "snapshot"`, I don't think we use the `publishVersion` property

Reply:
done in https://github.com/ACINQ/lightning-kmp/pull/231/commits/4241e85f20e94a0c6a92c5b87a2cb5ee5b2bdb38